.github/workflows/clients-java.yml

```diff
@@ -22,6 +15,8 @@ jobs:
 
     steps:
     - uses: actions/checkout@v3
+      with:
+        submodules: recursive
```

Comment:
This is on by default IIRC

Reply:
They changed the default in v3 I think - https://github.com/actions/checkout says it defaults to false